Extension/src/githubAPI.ts

```diff
@@ -201,7 +201,7 @@ export function getTargetBuild(builds: Build[], userVersion: PackageVersion, upd
         // check if the assetes are available (insider)
         useBuild = isBuild;
     } else if (updateChannel === 'Default') {
-        needsUpdate = function(installed: PackageVersion, target: PackageVersion): boolean { return installed.isGreaterThan(target); };
+        needsUpdate = function(installed: PackageVersion, target: PackageVersion): boolean { return target.isGreaterThan(installed); };
```

Comment:
I think this needs to be reverted (and a comment should be added).  When this function is called with the "Default" channel it's because the channel used to be "Insiders".  That means the user is leaving the insiders channel and needs to downgrade back to the latest release build.  So a comment here stating that the user is exiting the insiders channel and needs to downgrade would help us avoid changing this in the future.